lucene/core/src/java/org/apache/lucene/store/Directory.java

```diff
@@ -165,4 +165,13 @@ public void copyFrom(Directory from, String src, String dest, IOContext context)
    * @throws AlreadyClosedException if this Directory is closed
    */
   protected void ensureOpen() throws AlreadyClosedException {}
+  
```

Comment:
I think this means that org.apache.solr.store.blockcache also needs to move?  It's only used by the HDFS stuff, so far as I can tell.